### PR TITLE
chore(release): prepare release notes for OSS 23.04

### PR DESCRIPTION
**Pull request by tuntoja**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [ ] 22.10.x (staging)
- [x] 23.04.x (staging)
- [ ] Cloud (staging)
- [ ] Monitoring Connectors (staging)
- [ ] 23.10.x (next)

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.md [hunk 1]
```diff
@@ -18,6 +18,34 @@ Retrouvez plus de détails sur la version 23.04 dans notre [post de blog](https:
...
+- [API] Improved error codes and messages for the renewing password endpoint.
+- [Authentication] Fixed custom endpoint call with OpenId Connect.
+- [Authentication] fixed HTTP method for custom endpoints for OpenId.
+- [Authentication] Removed password expiration policy for LDAP authentication.
+- [Configuration] Fixed a bug in trap relations with services by host groups.
+- [Core] Improved the display of error messages regarding connection to DBMS.
+- [Install] Fixed snmptrapd configuration for Debian.
+- [LDAP] Fixed LDAP groups listing in contactgroups form.
+- [ResourcesStatus] Fixed a filtering issue for hosts in pending state.
+- [RessourcesStatus] Fixed h.name filter not returning hosts.
```
Comment (mdebauchez) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.md:43]: It's [ResourcesStatus] and not [RessourcesStatus]

File: versioned_docs/version-23.04/releases/centreon-os.md [hunk 2]
```diff
@@ -19,6 +19,34 @@ Read more about version 23.04 in our [blog post](https://www.centreon.com/centre
...
+- [API] Improved error codes and messages for the renewing password endpoint.
+- [Authentication] Fixed custom endpoint call with OpenId Connect.
+- [Authentication] fixed HTTP method for custom endpoints for OpenId.
+- [Authentication] Removed password expiration policy for LDAP authentication.
+- [Configuration] Fixed a bug in trap relations with services by host groups.
+- [Core] Improved the display of error messages regarding connection to DBMS.
+- [Install] Fixed snmptrapd configuration for Debian.
+- [LDAP] Fixed LDAP groups listing in contactgroups form.
+- [ResourcesStatus] Fixed a filtering issue for hosts in pending state.
+- [RessourcesStatus] Fixed h.name filter not returning hosts.
```
Comment (mdebauchez) [inline versioned_docs/version-23.04/releases/centreon-os.md:44]: It's [ResourcesStatus] and not [RessourcesStatus]

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.md [hunk 3]
```diff
@@ -18,6 +18,34 @@ Retrouvez plus de détails sur la version 23.04 dans notre [post de blog](https:
...
+- [UI] Fixed a bug that prevented non-admin users to select host groups in some areas of Centreon (eg. Host Discovery mappers).
+- [UI] Fixed refresh icon positionnement in "Administration > ACL > Reload ACL" menu.
+- [UI] Unifomized buttons size on legacy pages.
```
Comment (mdebauchez) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.md:46]: Uniformized and not Unifomized

File: versioned_docs/version-23.04/releases/centreon-os.md [hunk 4]
```diff
@@ -19,6 +19,34 @@ Read more about version 23.04 in our [blog post](https://www.centreon.com/centre
...
+- [UI] Fixed a bug that prevented non-admin users to select host groups in some areas of Centreon (eg. Host Discovery mappers).
+- [UI] Fixed refresh icon positionnement in "Administration > ACL > Reload ACL" menu.
+- [UI] Unifomized buttons size on legacy pages.
```
Comment (mdebauchez) [inline versioned_docs/version-23.04/releases/centreon-os.md:47]: Uniformized and not Unifomized

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.md [hunk 5]
```diff
@@ -18,6 +18,34 @@ Retrouvez plus de détails sur la version 23.04 dans notre [post de blog](https:
...
+- [Administration] Added button to unblock users through UI (for local authentication).
+- [Authentication] Fixed custom endpoint definition for OpenID Connect.
+- [Install] Removed PHP warning during installation wizard.
+- [Packaging] Improved default configuration for Debian packages.
+
+#### Bug fixes
+
+- [API] Improved error codes and messages for the renewing password endpoint.
+- [Authentication] Fixed custom endpoint call with OpenId Connect.
+- [Authentication] fixed HTTP method for custom endpoints for OpenId.
```
Comment (mdebauchez) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.md:36]: The first letter is not capitalized

File: versioned_docs/version-23.04/releases/centreon-os.md [hunk 6]
```diff
@@ -19,6 +19,34 @@ Read more about version 23.04 in our [blog post](https://www.centreon.com/centre
...
+- [Administration] Added button to unblock users through UI (for local authentication).
+- [Authentication] Fixed custom endpoint definition for OpenID Connect.
+- [Install] Removed PHP warning during installation wizard.
+- [Packaging] Improved default configuration for Debian packages.
+
+#### Bug fixes
+
+- [API] Improved error codes and messages for the renewing password endpoint.
+- [Authentication] Fixed custom endpoint call with OpenId Connect.
+- [Authentication] fixed HTTP method for custom endpoints for OpenId.
```
Comment (mdebauchez) [inline versioned_docs/version-23.04/releases/centreon-os.md:37]: The first letter is not capitalized

**Review by mdebauchez — CHANGES_REQUESTED** [review]

**Review by mdebauchez — DISMISSED** [review]

**Review by rsadik-ext — DISMISSED** [review]

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.md [hunk 7]
```diff
@@ -76,6 +104,18 @@ Compatibility with other 23.04 components.
 
 ## Centreon Gorgone
 
+### 23.04.2
+
+Release date : `soon`
+
+#### Enhancements
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.md:111]:
```suggestion
#### Enhancement
```

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.md [hunk 8]
```diff
@@ -76,6 +104,18 @@ Compatibility with other 23.04 components.
...
+
+- [Packaging] Improved default configuration for Debian packages.
+
+#### Bug fixes
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.md:115]:
```suggestion
#### Bug fix
```

File: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.md [hunk 9]
```diff
@@ -100,6 +140,14 @@ Release date: `April 26, 2023`
 
 ## Centreon DSM
 
+### 23.04.1
+
+Release date: `soon`
+
+#### Enhancements
```
Comment (smau-centreon) [inline i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.md:147]:
```suggestion
#### Enhancement
```

File: versioned_docs/version-23.04/releases/centreon-os.md [hunk 10]
```diff
@@ -77,6 +105,18 @@ Compatibility with other 23.04 components.
 
 ## Centreon Gorgone
 
+### 23.04.2
+
+Release date : `soon`
+
+#### Enhancements
```
Comment (smau-centreon) [inline versioned_docs/version-23.04/releases/centreon-os.md:112]:
```suggestion
#### Enhancement
```

File: versioned_docs/version-23.04/releases/centreon-os.md [hunk 11]
```diff
@@ -77,6 +105,18 @@ Compatibility with other 23.04 components.
...
+
+- [Packaging] Improved default configuration for Debian packages.
+
+#### Bug fixes
```
Comment (smau-centreon) [inline versioned_docs/version-23.04/releases/centreon-os.md:116]:
```suggestion
#### Bug fix
```

**Review by smau-centreon — CHANGES_REQUESTED** [review]

**Review by smau-centreon — APPROVED** [review]

**Review by jdelpierre — APPROVED** [review]

**Pull request merged**